### PR TITLE
Address feedback from review

### PR DESCRIPTION
**Pull request by atoader**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,19 +22,33 @@ Have you identified a reproducible problem in the AL compiler or developer tools
...
-- For suggestions for new static analysis rules or improvements to existing static analysis rules, use [Business Central Ideas](https://aka.ms/bcideas).
-- For issues related to the extensibility of applications authored by Microsoft, such as the `System Application` or the `Base Application`, use the [ALAppExtensions](https://github.com/Microsoft/ALAppExtensions) repository.
-- For issues concerning versions of Business Central that are in mainstream support you should use the [support channel](#support).
+#### **Security vulnerabilities**
+Please follow the instructions in this [document](SECURITY.MD) to ensure that the issue is handled with the highest priority.
+
+#### **Bugs**
+For bugs in the latest version of the AL Language extension for Visual Studio Code or any of the static analysis tools that accompany the AL Language extension, use the current [GitHub repository](https://github.com/microsoft/al). You should always use the latest version of the AL Language extension when targeting any version of Business Central.
```
Comment (qutreson) [inline CONTRIBUTING.md:29]: We should clarify what the latest version means. We should write _the latest AL Language extension for Visual Studio from the Visual Studio Code marketplace_. I know it's long, but it's unclear what the latest version means

File: .github/ISSUE_TEMPLATE/al-developer-experience-bug.md [hunk 2]
```diff
@@ -13,6 +13,9 @@ Read the contributing https://github.com/microsoft/AL/blob/master/CONTRIBUTING.M
 Please follow this template in order for our developers to investigate your issue efficiently.
 
 Do not edit or remove the titles; e.g. "Descibe the bug".
```
Comment (NKarolak) [inline .github/ISSUE_TEMPLATE/al-developer-experience-bug.md:15]: BTW there's a typo: "Desc**r**ibe the bug" :-)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,19 +22,33 @@ Have you identified a reproducible problem in the AL compiler or developer tools
...
+For new features in AL/Visual Studio Code language and tools, as well as Business Central in general, use [Business Central Ideas](https://aka.ms/bcideas). 
+
+For suggestions for new static analysis rules or improvements to existing static analysis rules, use [Business Central Ideas](https://aka.ms/bcideas).
+
+Posting your suggestion on [Business Central Ideas](https://aka.ms/bcideas) will allow other members of the community to vote on your idea and will allow us to prioritize based accordingly.
 
 ### Look for an existing issue
 
-Before you create a new issue, please do a search in [open issues](https://github.com/microsoft/AL/issues) to see if the issue or feature request has already been filed.
+Before you create a new issue, please do a search in [open issues](https://github.com/microsoft/AL/issues) to see if the issue or feature request has already been filed, and add to that instead.
```
Comment (qutreson) [inline CONTRIBUTING.md:47]: Should this be more explicit? If this issue has already been filed, contribute to the existing issue.

Comment (qutreson) [inline CONTRIBUTING.md:47]: We should not say _or feature request_ as we do not accept them anymore

Comment (qutreson) [reply CONTRIBUTING.md:47]: Actually, we are developing this point below. So maybe we should just remove what you added

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -43,6 +59,10 @@ If you cannot find an existing issue that describes your bug, create a new issue
 
 File a single issue per problem. Do not enumerate multiple bugs or feature requests in the same issue.
 
+Use the latest version of the AL language extension from the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=ms-dynamics-smb.al) or shipped as part of the [AL Developer Preview builds](README.md#al-developer-preview-builds) for Dynamics 365 Business Central.
+
+Switch off all extensions except the AL Language extension.
```
Comment (qutreson) [inline CONTRIBUTING.md:64]: The VS Code term is disable.

Disable all extensions installed in Visual Studio Code except the AL Language extension.

File: .github/ISSUE_TEMPLATE/al-developer-experience-bug.md (hunk 2 above)
Comment (atoader) [reply .github/ISSUE_TEMPLATE/al-developer-experience-bug.md]: Thanks! :)

**Pull request merged**